### PR TITLE
text, get_attribute() example added

### PR DESCRIPTION
**Pull request by MaBlaGit**
Hi,
I added new test, which return tag text and text of selected tags attribute.
Please review my code;)

### CODE REVIEW

**Comment by bajaaaa** [comment]
HI. Please explain more clearly what kind of text is returned by _get_element_text_

**Comment by MaBlaGit** [comment]
Ok, I'll explain that ASAP ;P

**Comment by MaBlaGit** [comment]
OK, I fixed comments, The are more clearer now, check it;)

File: simple_sel_excercises.py [hunk 1]
```diff
@@ -35,11 +35,11 @@ def test_get_text(self):
         name = "courses"
 
         # steps
-        # find text on an element
+        # property 'text' returns text located between tags
```
Comment (bajaaaa) [inline simple_sel_excercises.py]: Ok. I can accept this conditionally.

**Pull request merged**